core/src/term/mod.rs

```diff
@@ -1962,6 +1962,29 @@ pub mod make {
     pub fn integer(n: impl Into<i64>) -> RichTerm {
         Term::Num(Number::from(n.into())).into()
     }
+
+    pub fn static_access<I, S, T>(record: T, fields: I) -> RichTerm
+    where
+        I: IntoIterator<Item = S>,
+        I::IntoIter: DoubleEndedIterator,
+        S: Into<Ident>,
+        T: Into<RichTerm>,
+    {
+        let mut term = record.into();
+        for f in fields.into_iter() {
+            term = make::op1(UnaryOp::StaticAccess(f.into()), term);
+        }
+        term
+    }
+
+    pub fn static_access_<I, S>(record: S, fields: I) -> RichTerm
```

Comment:
I'm not too fond of naming a public symbol with a trailing `_`. Maybe something like `static_access_var`, which better reflects the semantics?

Tangentially, static access is not a great name, and inherited from the very first prototype, but it's probably not the role of this PR to change that :) I would vouch for something with `field` in it, at least

Reply:
In the end, I wonder if we even need this variant. `make::var` is like 9 characters, and if it's really heavily used somewhere (JSON schema converter), maybe the consuming crate can span its own `static_access_var`?

Reply:
Thinking about it again, I think you're right that `static_access_` isn't that useful. I'll get rid of it and just keep the other variant.